src/app.global.scss

```diff
@@ -63,6 +63,7 @@ a {
 .gc360_header {
   background-color: var(--mui-palette-primary-main);
   color: var(--mui-palette-primary-contrastText);
+  border-radius: 4px;
```

Comment:
Is this change relevant to the topic of this PR?

Reply:
Yes, because I updated the Schedule Panel to use this site-wide header style instead of a component-specific copy. The site-wide style typically has (almost) this border-radius regardless, because it's typically used with the CardHeader component. That border radius wasn't present here because the actual header is an AccordionSummary, not a CardHeader.

I have now gone in and removed the defunct header style from the component, as well fixing the border radius so that it's only applied to the top corners.